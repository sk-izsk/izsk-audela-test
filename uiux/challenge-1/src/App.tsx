import React from 'react';
import { createUseStyles } from 'react-jss';
import { CustomTheme } from './theme/theme';

const useStyles = createUseStyles((theme: CustomTheme) => ({
  '@global': {
    body: {
      padding: 0,
      margin: 0,
      backgroundColor: theme.colors.primaryColor,
      fontFamily: theme.fontProperties.fontFamily,
      scrollBehavior: 'smooth',
    },
  },
  loaderContainer: {
    display: 'flex',
    width: '100vw',
    height: '100vh',
    justifyContent: 'center',
    alignItems: 'center',
  },
}));

const App = () => {
  useStyles();
  return <div>hello</div>;
};

export { App };
