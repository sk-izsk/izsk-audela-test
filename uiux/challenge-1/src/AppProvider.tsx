import React from 'react';
import { ThemeProvider } from 'react-jss';
import { theme } from './theme';

export interface AppProviderProps {}

const AppProvider: React.FC<AppProviderProps> = ({ children }) => {
  return (
    <ThemeProvider theme={theme}>
      {/* <Provider store={store}> */}
      {children}
      {/* </Provider> */}
    </ThemeProvider>
  );
};

export { AppProvider };
