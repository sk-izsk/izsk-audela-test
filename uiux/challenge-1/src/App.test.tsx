import { render } from '@testing-library/react';
import React from 'react';
import { App } from './App';
import { AppProvider } from './AppProvider';

test('App renders correctly', () => {
  const { asFragment } = render(
    <AppProvider>
      <App />
    </AppProvider>,
  );
  expect(asFragment()).toMatchSnapshot();
});
