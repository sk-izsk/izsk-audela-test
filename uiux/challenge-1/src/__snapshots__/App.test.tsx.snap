// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<DocumentFragment>
  <div
    class="mainContainer-0-2-1"
  >
    <div
      class="_2xbD- inputContainer-0-2-3 textField-0-2-2"
    >
      <div
        class="_fWLwS"
      >
        <label
          class="_Mk5N_"
          for="_77467_"
        >
          <div
            class="_3yPwv _2BgaH _1QRSN _1uvOH "
          />
        </label>
        <input
          class="_U6nBC _2nHt_ "
          id="_77467_"
          style="width: 800px; min-height: undefinedpx;"
          type="text"
          value=""
        />
        <div
          class="_3t8tT"
        >
          <div
            class="_3yPwv _2HbLl _1QRSN _1uvOH _2Lw0g"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
